osmosis-1/osmosis-1.assetlist.json

```diff
@@ -7253,6 +7253,49 @@
         "ATOM:944"
       ]
     },
+    {
```

Comment:
please remove this, as it will be generated more accurately--we only need the zone file entry

Reply:
done